src/pages/Dashboard.tsx

```diff
@@ -1,39 +1,39 @@
-import React, { useState } from 'react';
-import UserHeader from '../components/Dashboard/UserHeader';
-import StatusPanel from '../components/Dashboard/StatusPanel';
-import DataDisplay from '../components/Dashboard/DataDisplay';
-import './Dashboard.css';
-
-const Dashboard: React.FC = () => {
-  const [selectedMetric, setSelectedMetric] = useState('blood-pressure');
```

Comment:
Metric seems the kind of data that need's to be shared across multiple components, like charts, tables and all sorts of things need to work with it. It might be interesting placing it in a context.